### PR TITLE
CP-15132: Package daemons for systemd

### PR DESCRIPTION
**Pull request by simonjbeaumont**

### CODE REVIEW

**Comment by simonjbeaumont** [comment]
@johnelse one more commit here despite the Reviewed label. Can you ACK?

File: SOURCES/xcp-rrdd-plugins.service [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=XCP RRD daemon plugins
+After=network-online.target xenstored.service xcp-rrdd.service
+Conflicts=shutdown.target
```
Comment (johnelse) [inline SOURCES/xcp-rrdd-plugins.service]: Is this needed? gpumon doesn't have it.

Comment (simonjbeaumont) [inline SOURCES/xcp-rrdd-plugins.service]: Yes. This is from the autogenerated service file from the systemd-sysv-generator. It's still type forking which gpumon is not.

Comment (johnelse) [inline SOURCES/xcp-rrdd-plugins.service]: Ok fine :-)

**Pull request merged**